tests/extension.test.ts

```diff
@@ -14,59 +14,119 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 const vscode = require('./__mocks__/vscode');
-const cloudbuild = require('./__mocks__/cloudbuild');
-import {activate, statusBar, deactivate} from '../src/extension';
-import { StatusBarAlignment } from 'vscode';
+import * as cloudBuild from '../src/cloudbuild';
+import {activate, statusBar, deactivate, refreshTicker} from '../src/extension';
+import {StatusBarAlignment} from 'vscode';
+import {GitRepo} from '../src/git';
 
 describe('extension', () => {
-   let context: any;
+  let context: any;
+  let fetchBuildSpy: any;
+  let getCurrentBranchSpy: any;
 
-   beforeEach(() => {
-      vscode.commands.registerCommand.mockReset();
-      vscode.window.createStatusBarItem.mockReset();
-      vscode.window.createStatusBarItem.mockImplementation((align:StatusBarAlignment, prio:number) => {
-         return {
-            alignment: align,
-            priority: prio,
-            text: '',
-            tooltip: '',
-            show: jest.fn()
-         };
-      });
-      cloudbuild.fetchBuilds.mockReset();
-      context = {
-         subscriptions: {
-            push: jest.fn()
-         }
+  beforeEach(() => {
+    vscode.commands.registerCommand.mockReset();
+    vscode.window.createStatusBarItem.mockReset();
+    vscode.window.createStatusBarItem.mockImplementation((align: StatusBarAlignment, prio: number) => {
+      return {
+        alignment: align,
+        priority: prio,
+        text: '',
+        tooltip: '',
+        show: jest.fn(),
       };
-   });
+    });
+    fetchBuildSpy = jest.spyOn(cloudBuild, 'fetchBuilds').mockResolvedValue([]);
+    getCurrentBranchSpy = jest.spyOn(GitRepo.prototype, 'getCurrentBranch').mockResolvedValue('develop');
```

Comment:
Why using develop here ? I mean we mock a Git repo here, and master is a default branch always present in Git.